src/styles/style.css

```
@@ -94,60 +99,61 @@ ul {
 
-#searchInput {
-  padding-block: 0.5rem;
-  background-color: transparent;
+#search-input {
+  width: 200px;
+  transition: 1s linear;
   color: #fff;
-  border: none;
-  outline: none;
+  background: #1a1818b0;
+  color: #fff;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)